presidio-anonymizer/presidio_anonymizer/services/aes_cipher.py

```diff
@@ -0,0 +1,58 @@
+from typing import Tuple
+
+from Crypto.Util.Padding import pad, unpad
+
+import base64
+from Crypto import Random
+from Crypto.Cipher import AES
+
+
+class AESCipher:
+    """Advanced Encryption Standard (aka Rijndael) en/decryption in CBC mode."""
+
+    def __init__(self, key: bytes) -> None:
+        """
+        Instantiate AESCipher.
+
+        :param key: AES key in bytes
+        """
+        self.key = key
+
+    def encrypt(self, text: str) -> str:
```

Comment:
How about having the ket be part of the encrpt/decrypt API, this way the same ciper class can be re-used?

Reply:
Exactly why I chose to put it as a class instance.. makes more sense when you use as package, downside is instantiating it multiple times when using as a service. What do you think is better?
```python
aes = AESCipher(key)
aes.encrypt("text")
aes.decrypt("text")
```

Reply:
even when used as a package, users do not use it directly, but inderctly using the Encrypt anonymizer. This mean that every anonymizer call will instatiate a new Cipher object, To be more alighn with the rest of the implemetation I would go with aving a stateless approach, where the encrypt/decrypt methods recieve the key as an argument.

Reply:
Makes sense, will push a change

Reply:
@navalev Done

Reply:
approved